webapp/src/constants/action_types.js

```diff
@@ -1,7 +1,7 @@
 import {PLUGIN_NAME} from './manifest';
```

Comment:
`PLUGIN_NAME` is hardcoded in this imported manifest file. Can we instead have it derived from the generated manifest file? Non-blocking for approval

Reply:
I'm replacing `PLUGIN_NAME` from `constants/manifest.js` with `id` from `webapp/src/manifest.js`